docker/local/polygon-edge.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+set -e
+
+POLYGON_EDGE_BIN=./polygon-edge
+
+case "$1" in
+
+   "init")
+      echo "Generating secrets..."
+      node1id=$("$POLYGON_EDGE_BIN" secrets init --data-dir data-1 | grep Node | awk -F ' ' '{print $4}')
```

Comment:
This works fine, but you could use `--json` flag, which will return `json` output so that you wouldn't need to do all this awk shenanigans. 
Not a big issue, but if something changes in the output format, you'd need to rewrite this command.

Reply:
Good point, done